--- packages/cercopitheque/includes/YesWikiInit.old.php	2020-12-09 13:49:55.307000000 +0100
+++ packages/cercopitheque/includes/YesWikiInit.php	2020-12-09 13:57:00.712958361 +0100
@@ -266,17 +266,15 @@
     {
         // configuration du cookie de session
         // determine le chemin pour les cookies
-        $a = parse_url($this->config['base_url']);
-        $CookiePath = dirname($a['path']);
+        $parsedUrl = parse_url($this->config['base_url']);
+        $CookiePath = '/';
+        if (isset($parsedUrl['path'])) {
+            $CookiePath = $parsedUrl['path'] . '/';
+        }
 
         // Fixe la gestion des cookie sous les OS utilisant le \ comme separteur de chemin
         $CookiePath = str_replace('\\', '/', $CookiePath);
 
-        // ajoute un '/' terminal sauf si on est a la racine web
-        if ($CookiePath != '/') {
-            $CookiePath .= '/';
-        }
-
         // test if session exists, because the wiki object is instanciated for every plugin
         if (!isset($_SESSION)) {
             $a = session_get_cookie_params();
