--- packages/cercopitheque/includes/YesWikiInit.php	2020-12-07 17:53:02.728244472 +0100
+++ packages/cercopitheque/includes/YesWikiInit.php	2020-12-07 17:53:12.971625201 +0100
@@ -267,7 +267,7 @@
         // configuration du cookie de session
         // determine le chemin pour les cookies
         $a = parse_url($this->config['base_url']);
-        $CookiePath = dirname($a['path']);
+        $CookiePath = $a['path'];
 
         // Fixe la gestion des cookie sous les OS utilisant le \ comme separteur de chemin
         $CookiePath = str_replace('\\', '/', $CookiePath);
